### PR TITLE
io.dbeaver.DBeaverCommunity.Client.pgsql extension for DBeaver

### PR DESCRIPTION
**Pull request by krashekspres**

### CODE REVIEW

File: io.dbeaver.DBeaverCommunity.Client.pgsql.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "app-id": "io.dbeaver.DBeaverCommunity.Client.pgsql",
+  "runtime": "io.dbeaver.DBeaverCommunity",
+  "sdk": "org.freedesktop.Sdk//18.08",
```
Comment (nedrichards) [inline io.dbeaver.DBeaverCommunity.Client.pgsql.json]: could you show this as a separate runtime-version please?

**Comment by nedrichards** [comment]
this needs a separate flathub.json file that limits it only x86-64 like the parent project. https://github.com/flathub/io.dbeaver.DBeaverCommunity/blob/master/flathub.json

**Comment by nedrichards** [comment]
Can you also include appdata please, then software stores will be able to correctly link and show these extensions to be installed inline.

**Comment by bilelmoussaoui** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.pgsql

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.pgsql

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.pgsql

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.pgsql

**Comment by krashekspres** [comment]
bot, build io.dbeaver.DBeaverCommunity.Client.pgsql

**Comment by krashekspres** [comment]
@nedrichards is this now ready, or do we need any more changes?

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.dbeaver.DBeaverCommunity.Client.pgsql

**Pull request closed without merging**